docs/concepts/projects.md

```diff
@@ -39,6 +42,72 @@ dependency list from the command line with `uv add` and `uv remove`. uv also sup
 
     See the official [`pyproject.toml` guide](https://packaging.python.org/en/latest/guides/writing-pyproject-toml/) for more details on getting started with a `pyproject.toml`.
 
+## Creating projects
+
+uv supports initializing a project with `uv init`. By default, uv will create a project in the
+working directory. Projects can be created in a target directory by providing a name, e.g.,
+`uv init foo`. If there's already a project in the target directory, i.e., there's a
+`pyproject.toml`, uv will exit with an error.
+
+By default, uv will create a project for an [application](#applications). However, the `--lib` flag
+can be used to create a project for a [library](#libraries).
+
+## Project types
+
+uv groups projects into two types: **applications** and **libraries**.
+
+### Applications
+
+Application projects are suitable for web servers, scripts, and command-line interfaces.
+
+Application projects have the following traits:
+
+- A build backend is not defined.
```

Comment:
But this isn't _required_ right?

Reply:
Well.. I wrote it that way at first but that gets confusing. I think we have

- Applications: no build backend
- Libraries: yes build backend
- Other: you set `package` manually